content/influxdb/v2.0/reference/api/influxdb-1x/dbrp.md

```diff
@@ -47,9 +47,6 @@ the DBRP mapping service checks for a bucket mapped to the database and retentio
     - **database** exists (without a specified retention policy), the default
       database retention policy is used, a DBRP mapping is added to the bucket,
       and data is written to the bucket.
-- If no matching bucket is found, a new **database/retention-policy** bucket is
```

Comment:
(AFAIK) the point above is also outdated, DBRP mappings aren't auto-created in any situation.

This change is unique to OSS, so the top of the section that says `When writing data to InfluxDB Cloud and InfluxDB OSS 2.0` will need to change. I'm not sure how splits are handled in situations like this...

Reply:
Thanks @danxmoran , updated both.